12-Principles-Of-Agile-Manifesto.md

```diff
@@ -0,0 +1,29 @@
+# 12 Principles Behind The Agile Manifesto
+
+### We follow these principles:
+
+1. Our highest priority is to satisfy the customer through early and continuous delivery of valuable software.
+
+2. Welcome changing requirements, even late in development. Agile processes harness change for the customer's competitive advantage.
```

Comment:
This was copied, it is open for all, directly from the source. Would it help, in your opinion, if I added the link to the original? My first thought was to keep them on our wiki but I can see linking to the source as a reference as well.